src/components/Modals/ExecuteModal.js

```diff
@@ -308,4 +332,5 @@ ExecuteModal.propTypes = {
   runRemediation: PropTypes.func,
```

Comment:
We can remove `runRemediation` from here since ExecuteModal has no such prop anymore